### ISSUE TITLE
Intra-abdominal adhesion

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6181]
**Preferred term label:**
Intra-abdominal adhesion

**Synonyms**
Abdominal adhesion

**Definition (free text, please give PubMed ID)**
Normally separated organs are attached to each other and can cause major problems for the affected patients by giving rise to small bowel obstruction, chronic pelvic pain, dyspareunia, infertility, and higher complication rates in subsequent operations. Intra-abdominal adhesions arise from aberrant peritoneal wound healing processes, so any mesothelial damage by surgical trauma or bacterial inflammation can lead to their formation.
PMID: 21116396 

**Parent term (use hpo.jax.org/app)**
Abnormal intestine morpholoy

**Comment by pnrobinson** [comment]
Adding, but I will use this definition:

Fibrous bands (i.e., bands of scar-like tissue) that span two or more intra-abdominal organs and/or the inner abdominal wall (i.e. peritoneal membrane). 

Comment: Abdominal adhesions typically form after abdominal surgery but may also occur secondary to inflammatory conditions of the abdomen in the absence of prior abdominal surgery or as a sequela of abdomino-pelvic radiation. Although the majority of patients with intra-abdominal adhesions remain asymptomatic, a clinically significant subset of patients will develop “adhesive disease”, a symptomatic state ranging from mild and/or vague to highly distressing and even life-threatening symptoms

HP_0033134
